src/storage.ts

```diff
@@ -0,0 +1,77 @@
+import { DataStoreLevel, EventLogLevel, MessageStoreLevel, DwnConfig } from '@tbd54566975/dwn-sdk-js';
+import { SqliteDialect, MysqlDialect, PostgresDialect, Dialect, MessageStoreSql, DataStoreSql, EventLogSql } from '@tbd54566975/dwn-sql-store';
+
+import Database from 'better-sqlite3';
+import { createPool as MySQLCreatePool } from 'mysql2';
+import pg from 'pg';
+import Cursor from 'pg-cursor';
+
+const storageTypes = {
+  LEVEL    : 'level',
+  SQLITE   : 'sqlite',
+  MYSQL    : 'mysql',
+  POSTGRES : 'postgres'
+};
+
+function createDWNFromDB(db: Dialect): DwnConfig {
+  const messageStore = new MessageStoreSql(db);
+  const dataStore = new DataStoreSql(db);
+  const eventLog = new EventLogSql(db);
+  return { eventLog, dataStore, messageStore };
+}
+
+export function getDWNConfig(): DwnConfig {
+  // Initialize storage based on DWN_STORAGE environment variable
+  const storageURLstring = process.env.DWN_STORAGE || 'level://data';
+  const storageURL = new URL(storageURLstring);
+  const scheme = storageURL.protocol.slice(0, -1); // protocol always ends with ':'
+
+  switch(scheme) {
+  case storageTypes.LEVEL: {
+    const storageDir = storageURL.host + storageURL.pathname;
+    console.log('using leveldb storage in', storageDir);
```

Comment:
do you want to leave this?

Reply:
oh i suppose not. I was going to log for all of them, but that would be difficult to sanitize for the databases that might have passwords. I have removed it.